src/environments/environment.prod.ts

```diff
@@ -25,7 +25,7 @@ export const authConfig: AuthConfig = {
   issuer: '${AUTH_URL}/realms/backbase',
 
   // URL of the SPA to redirect the user to after login
-  redirectUri: '${PROTOCOL}//${HOSTNAME}:${PORT}${PATHNAME}${AUTH_LANDING_PAGE}',
+  redirectUri: '${PROTOCOL}//${HOSTNAME}:${PORT}${BASE_HREF}${PATHNAME}${AUTH_LANDING_PAGE}',
```

Comment:
can you change this to use `window.location` while you are at it? That way it does not need any interpolation